internal/services/containers/kubernetes_cluster_resource_test.go

```diff
@@ -202,6 +202,34 @@ func TestAccKubernetesCluster_edgeZone(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesCluster_updateNetworkProfileOutboundType(t *testing.T) {
```

Comment:
Would you mind removing the test `TestAccKubernetesCluster_outboundTypeUpdate` which also tests the update of the outbound type but is less comprehensive.